### PR TITLE
Fixed and Updated french localization

### PR DESCRIPTION
**Pull request by Cimmerian-Iter**

### CODE REVIEW

**Comment by CharlieWhile13** [comment]
Please add the following keys 
```xml
"Auto_Complete_Queue" = //"Auto Complete Queue";
"Enable_Analytics" = //"Enable Analytics";
"View_Depiction" = //"View Depiction";
```
and remove the keys
```xml
"Package_Filter_User" 
"Package_Filter_PowerUser" 
"Package_Filter_Dev"
```

**Comment by Cimmerian-Iter** [comment]
Do I translate the string after // or no?

**Comment by CharlieWhile13** [comment]
Yes, sorry. Translate and remove the comment

**Comment by althiometer** [comment]
> "Dark_Mode_Toggle" = "Mode Nuit";
"Mode Sombre" might be better ngl

**Comment by Cimmerian-Iter** [comment]
ok done

**Review by CharlieWhile13 — APPROVED** [review]

**Review by CharlieWhile13 — APPROVED** [review]

File: Sileo/Sileo/fr.lproj/Localizable.strings [hunk 1]
```diff
@@ -96,46 +96,65 @@
...
 "Settings_Translations_Heading" = "Traductions";
 
 "Settings_Payment_Provider.Purchases_Heading" = "Achats";
 "Settings_Payment_Provider.No_Purchases" = "Aucun achat";
 
+"Settings" = "Paramètres";
+"Settings_Page_Title" = "Paramètres";
+"Dark_Mode_Toggle" = "Mode Sombre";
+"Theme" = "Thème";
+"Tint_Color" = "Couleur de la teinte";
```
Comment (06Games) [inline Sileo/Sileo/fr.lproj/Localizable.strings:124]: Maybe "Couleur d'accentuation" would be better

File: Sileo/Sileo/fr.lproj/Errors.strings [hunk 2]
```diff
@@ -21,33 +23,32 @@
...
+"FixingDpkg.Title" = "Interruption de Dpkg";
+"FixingDpkg.Body" = "Dpkg a été interrompu. Veuillez patienter pendant que nous essayons de réparer le problème.";
 
 "Invalid_Data" = "Données non-valides";
 "Try_Later" = "Ré-essayer plus tard, s'il vous plaît.";
 
 "Sileo_Update_Required.Title" = "Une mise à jour de Sileo est requise";
-"Featured_Requires_Sileo_Update" = "Une mise à jour de Sileo est requise pour voir l'onglet de paquets mise en vedette. Cet onglet sera possiblement rendu incorrectement.";
-"Depiction_Requires_Sileo_Update" = "Une mise à jour de Sileo est requise pour voir cette page. La page sera possiblement rendu incorrectement.";
+"Featured_Requires_Sileo_Update" = "Une mise à jour de Sileo est requise pour voir l'onglet "Mise en vedette". La page a des chances d'être rendue incorrectement.";
```
Comment (06Games) [inline Sileo/Sileo/fr.lproj/Errors.strings:43]: I think "affichée" would be more appropriate than "rendue"

File: Sileo/Sileo/fr.lproj/Localizable.strings [hunk 3]
```diff
@@ -96,46 +96,65 @@
...
-"Dangerous_Repo.Body" = "Vous essayez d'installer une source qui est connue être dangereuse pour des raisons qui peuvent inclure la piratie, l'hébergement de contenu illégale ou des virus.\n\nLes sources indiqués être dangereuses distribuent souvent des paquets ayant des effets inattendues, soit en forme de paquets payés rendus gratuits, ou d'autres paquets malicieux. Ceux-ci peuvent avoir des effects permanents sur votre appareil, et peuvent même réduire votre capacité pour installer un jailbreak. Nous recommendons que vous agissez avec prudence pour éviter des logiciels dangereux.";
+"Dangerous_Repo.Body" = "Vous essayez d'installer une source qui est connue pour être dangereuse (hébergeant des paquets piratés, du contenu illégal ou des virus).\n\nLes sources indiqués être dangereuses distribuent souvent des paquets ayant des effets inattendues, soit sous forme de paquets payant rendus gratuits, ou d'autres paquets malicieux. Ceux-ci peuvent avoir des effects permanents sur votre appareil et votre vie privée, et peuvent même altérer le fonctionnement d'un jailbreak. Nous vous recommandons d'agir avec prudence afin d'éviter des logiciels dangereux.";
```
Comment (06Games) [inline Sileo/Sileo/fr.lproj/Localizable.strings:145]: I think "indiqué**e**s **comme** dangereuses" would be better than "indiqués être dangereuses"

File: Sileo/Sileo/fr.lproj/Localizable.strings [hunk 4]
```diff
@@ -96,46 +96,65 @@
...
 "Cydia_Sign_In" = "Se connecter au Cydia Store.";
 "Payment_Provider_Signed_In" = "Connecté";
 "Payment_Provider_Signed_In_With_Name" = "Utilisateur connecté: %@"; // %@ = user's name
 "Payment_Provider_Sign_Out" = "Déconnexion";
 "Payment_Provider_Sign_Out_Confirm.Title" = "Déconnecter de %@"; // %@ = payment provider name
-"Payment_Provider_Sign_Out_Confirm.Body" = "Vous ne serez pas capable de télécharger vos paquets acheter avant de re-connecter.";
+"Payment_Provider_Sign_Out_Confirm.Body" = "Vous ne serez pas capable de récupérer vos paquets achetés avant de vous re-connecter.";
 
+"Show_Install_Details" = "Afficher les détails";
 "Show_Install_Details" = "Dévoiler les détails";
```
Comment (06Games) [inline Sileo/Sileo/fr.lproj/Localizable.strings:137]: I think this line should be deleted

File: Sileo/Sileo/fr.lproj/Errors.strings (hunk 2 above)
Comment (06Games) [reply Sileo/Sileo/fr.lproj/Errors.strings:43]: The quotes must also be prefixed with a backslash for the string to be correctly read

File: Sileo/Sileo/fr.lproj/Localizable.strings (hunk 1 above)
Comment (Cimmerian-Iter) [reply Sileo/Sileo/fr.lproj/Localizable.strings:124]: Yeah checked how mac os translated this in french, while I don't think it's better for the user better stick with the "official" translation

File: Sileo/Sileo/fr.lproj/Errors.strings (hunk 2 above)
Comment (Cimmerian-Iter) [reply Sileo/Sileo/fr.lproj/Errors.strings:43]: removed the quotes, they aren't necessary

File: Sileo/Sileo/fr.lproj/Localizable.strings (hunk 4 above)
Comment (Cimmerian-Iter) [reply Sileo/Sileo/fr.lproj/Localizable.strings]: yeah leftover I forgot to delete thanks

**Comment by CharlieWhile13** [comment]
Is this good to merge now?

**Comment by 06Games** [comment]
I don't see any other changes to make. So in my opinion, yes

**Review by Diatrus — APPROVED** [review]

**Pull request merged**